### PR TITLE
build package

### PR DESCRIPTION
**Pull request by moritzwilksch**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by moritzwilksch** [comment]
@conda-forge-admin, please rerender

**Comment by xhochy** [comment]
I wanted to debug this locally and thus added ARM architectures but for them, `pylance` is not available. Requested it via https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/5960

File: recipe/build.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+set -ex
+export OPENSSL_DIR=$PREFIX
+
+# Bundle all library licenses
+cargo-bundle-licenses \
+  --format yaml \
+  --output ${SRC_DIR}/THIRDPARTY.yml
+
+$PYTHON -m pip install --no-deps --ignore-installed python
```
Comment (isuruf) [inline recipe/build.sh:11]:
```suggestion
$PYTHON -m pip install --no-deps --ignore-installed ./python
```

**Comment by xhochy** [comment]
Confirmed locally that this works on `osx-64` and `osx-arm64`

**Comment by moritzwilksch** [comment]
should we consolidate this with #9 / #10?

**Comment by jan-janssen** [comment]
> should we consolidate this with #9 / #10?

I fixed all the issues in https://github.com/conda-forge/lancedb-feedstock/pull/12

**Comment by moritzwilksch** [comment]
superseded by #12

**Pull request closed without merging**